docs/technical_reference/unit_models/zero_order_unit_models/index.rst

```diff
@@ -1,5 +1,8 @@
 Zero-Order Unit Models
 ======================
+
+The YAML database files for the zero-order models are located in :code:`watertap/data/techno_economic/`. The YAML file name should match the associated model name without the ZO letters and use snake case style. For example, the YAML file for the :code:`DualMediaFiltrationZO()` zero-order model is named :code:`dual_media_filtration.yaml`.
+
```

Comment:
This might be a good place to tack on a disclaimer about the data, basically pointing out that they were mainly meant to serve as default values and should be taken with a grain of salt. We could also mention here that users are intended to supply their own data, after they get the hang of running models with defaults.

Reply:
This could be done here or in a subsequent PR. I just wanted to make note of it here before forgetting about it.

Reply:
Made some changes to address this. See recent commit.